docs/AzureBatch.md

```diff
@@ -2,43 +2,49 @@
-The process is designed as an [Azure Batch](https://docs.microsoft.com/en-us/azure/batch/) job that is scheduled weekly. This job starts the script `/ADBench/AzureBatch/runallscript.sh` on the pool node. This script clones the repository from GitHub, creates a Docker container, and runs all tools. After run completion the script runs graph creating and then stores result plots to the output directory from which they are uploaded to the [Azure Storage](https://docs.microsoft.com/en-us/azure/storage/).
+The process is designed as an [Azure Batch](https://docs.microsoft.com/en-us/azure/batch/) job that is scheduled weekly. This job starts the script `/ADBench/AzureBatch/runallscript.sh` on the pool node. This script clones the repository from GitHub, creates a Docker container, and runs all tools. After run completion the script runs graph creating and then stores the result plots to the output directory from which they are uploaded to the [Azure Storage](https://docs.microsoft.com/en-us/azure/storage/).
 
 ### Recalculating check
-If graphs have been already created for the last commit, then the script finishes and does nothing. To check that the last commit has been handled, the process stores its hash to the file called `last_commit.txt` located in the blob container root. Before the script starting the job downloads this file to the pool node. The script compares the file content with the latest repository master branch commit hash. In case of equality it finishes. Exit code the script returns in such a case is not zero because zero exit code is a signal to the job to upload result graphs to the storage. In case of hash difference the script write the current handled comit hash to the file and then the job uploads this file to the blob container.
+If graphs have been already created for the last commit, then the script finishes and does nothing. To check that the last commit has been handled, the process stores its hash to the file called `last_commit.txt` located in the blob container root. Before the script starting the job downloads this file to the pool node. The script compares the file content with the latest repository master branch commit hash. In case of equality it finishes. Exit code the script returns in such a case is not zero because zero exit code is a signal to the job to upload the result graphs to the storage. In case of hash difference the script writes the current handled comit hash to the file `last_commit.txt`, and then the job uploads this file to the blob container.
 
 ### The script
-`runallscript.sh` is a bash script that is designed for *Ubuntu 18.04*. It has descriptive comments, so, you can find more information about its behaviour in them. The script returns the following exit codes:
+`/ADBench/AzureBatch/runallscript.sh` is a bash script that is designed for *Ubuntu 18.04*. It has descriptive comments, so you can find more information about its behaviour in them. The script returns the following exit codes:
```

Comment:
Could we specify all paths from the top-level of the repository (and in particular not assume that the repository directory is called `ADBench`)?

Reply:
As you can see, paths have already been specified from the repository top-level. `/ADBench/AzureBatch/runallscript.sh` is located in the directory `ADBench/AzureBatch` inside the repository root, so `ADBench` is not the root of the repository

Reply:
Ah great, in that case I think it would be clearer to drop the leading `/` from the paths.

Reply:
Done